circles-app/src/lib/stores/wallet.svelte.ts

```
@@ -14,123 +14,116 @@ import { getCirclesConfig } from '$lib/utils/helpers';
+
+    signer.address = runner.address;
 
-    //TODO: cache environment on local storage
     const sdk = new Sdk(
-      restoredWallet as SdkContractRunner,
-      await getCirclesConfig(100n, environment.ring),
+      runner,
+      await getCirclesConfig(100n, rings),
     );
     circles.set(sdk);
+    let savedGroup = CirclesStorage.getInstance().group;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'savedGroup' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)